app/scripts/migrations/119.test.ts

```diff
@@ -0,0 +1,128 @@
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { migrate, version } from './119';
+
+const mockTimeStamp = 1716972230;
+jest.useFakeTimers().setSystemTime(new Date(mockTimeStamp));
+
+const oldVersion = 118;
+const newVersion = 119;
```

Comment:
Either get rid of this one, or use `const newVersion = version` (if you want to keep the "symmetry" between old/new in the constant name)